tgui/packages/tgui/interfaces/Secrets.js

```diff
@@ -0,0 +1,723 @@
+                content="Mass Remove Purrbation"
+                onClick={() => act("massremovepurrbation")} />
+            </NoticeBox>
+          </Flex.Item>
+        </Flex>
+      </Flex.Item>
+      <Flex.Item>
+        <Flex justify="space-between">
+          <Flex.Item width="240px" mr={0.25}>
+            <NoticeBox danger>
+              <Button
+                color="red"
+                align="center"
+                icon="flushed"
+                fluid
+                content="Fully Immerse Everyone"
+                onClick={() => act("massimmerse")} />
+            </NoticeBox>
+          </Flex.Item>
+          <Flex.Item grow={1} ml={0.25}>
+            <NoticeBox info>
+              <Button
+                color="blue"
+                align="center"
+                icon="sync-alt"
+                fluid
+                content="Shatter the Immersion"
+                onClick={() => act("unmassimmerse")} />
+            </NoticeBox>
+          </Flex.Item>
+        </Flex>
+      </Flex.Item>
+    </Flex>
+  );
+};
+
+export const Secrets = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    is_debugger,
+    is_funmin,
+  } = data;
+  const TAB2NAME = [
+    {
+      title: 'Debugging',
+      blurb: 'Where useless shit goes to die',
+      gauge: 5,
+      component: () => DebuggingTab,
+    },
+    {
+      title: 'Helpful',
+      blurb: 'Where fuckwits put logging',
+      gauge: 25,
+      component: () => HelpfulTab,
+    },
+    {
+      title: 'Fun',
+      blurb: 'How I ran an """event"""',
+      gauge: 75,
+      component: () => FunTab,
+    },
+    {
+      title: 'Only Fun For You',
+      blurb: 'How I spent my last day adminning',
+      gauge: 95,
+      component: () => FunForYouTab,
+    },
+  ];
+  const [
+    tabIndex,
+    setTabIndex,
+  ] = useLocalState(context, 'tab-index', 2);
+  const TabComponent = TAB2NAME[tabIndex-1].component();
+  return (
+    <Window
+      title="Secrets Panel"
+      width={500}
+      height={485}>
+      <Window.Content>
+        <Flex direction="column" height="100%">
+          <Flex.Item mb={1}>
+            <Section
+              title="Secrets"
+              buttons={(
+                <Fragment>
+                  <Button
+                    color="blue"
+                    icon="address-card"
+                    content="Admin Log"
+                    onClick={() => act("admin_log")} />
+                  <Button
+                    color="blue"
+                    icon="eye"
+                    content="Show Admins"
+                    onClick={() => act("show_admins")} />
+                </Fragment>)}>
+              <Flex
+                mx={-0.5}
+                align="stretch"
+                justify="center">
+                <Flex.Item bold>
+                  <NoticeBox color="black">
+                    {"The first rule of adminbuse is:\
+                    you don't talk about the adminbuse."}
```

Comment:
Do not use `{""}`, place text into the tag as is.

Reply:
Quotes and apostrophes throw lint errors when outside of ellipses,  which is why i did it. Would you know anything about that?

Reply:
Right... You can use `&apos;`.

Reply:
Ooooooooh